.github/workflows/autoupdate-sdk.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update SDK
+on:
+ schedule:
+   - cron: 30 8 * * MON
```

Comment:
we prefer Tuesdays as we usually have bank holidays on Monday

Reply:
```suggestion
   - cron: 30 8 * * TUE
```